tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -165,14 +202,20 @@ export interface Alerts {
   focusType: AlertType | null;
 }
 
-export interface Executions {
-  // Load state for the total number of top-level executions.
+/**
+ * Base interface shared between top-level and intra-graph executions.
+ *
+ * Supports paged lazy loading of digess (i.e., concise data objects
```

Comment:
s/digess/digest 

`Supports paged lazy loading of digest` doesn't read nicely properly in my head. It may be just me.

Reply:
Fixed the typo and tweaked the wording here for clarification.